docs/operator-manual/airgap/01-install.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_label: "Installation"
+title: ""
+---
+
+# Airgap installation
```

Comment:
nitpicking: we have to be consistent about how we refer to "Air gap": I can see we're using "airgap" and "air-gap".

I vote to use [`air gap`](https://en.wikipedia.org/wiki/Air_gap_(networking))

Reply:
done https://github.com/kubewarden/docs/pull/150/commits/4bb22cd9a36d2f12a445486e32ff690daa23b3bb